lib/router-config/src/usage_reporting.rs

```diff
@@ -0,0 +1,86 @@
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Deserialize, Serialize, JsonSchema, Clone)]
+#[serde(deny_unknown_fields)]
+pub struct UsageReportingConfig {
+    /// Your [Registry Access Token](https://the-guild.dev/graphql/hive/docs/management/targets#registry-access-tokens) with write permission.
+    pub token: String,
+    /// A target ID, this can either be a slug following the format “$organizationSlug/$projectSlug/$targetSlug” (e.g “the-guild/graphql-hive/staging”) or an UUID (e.g. “a0f4c605-6541-4350-8cfe-b31f21a4bf80”). To be used when the token is configured with an organization access token.
+    pub target_id: Option<String>,
```

Comment:
nit: this one can be validated during de-serialization as either `{string}/{string}/{string}` or `uuid`.

Reply:
Done 👍